IOS/SwiftSeedProject/SwiftSeedProject/Domain/Article+CoreDataProperties.swift

```diff
@@ -19,8 +19,8 @@ extension Article {
     @NSManaged public var author: String?
     @NSManaged public var title: String?
     @NSManaged public var detail: String?
-    @NSManaged public var url: String?
-    @NSManaged public var urlToImage: String?
+    @NSManaged public var url: URL?
```

Comment:
Yes, just to use URLs instead of Strings. I marked those attributes as Transformables